deployment/ansible/roles/raster-foundry.docker/templates/daemon.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+    "userns-remap": "{{ '' if docker_user == 'vagrant' else 'jenkins' }}",
+    "default-ulimits": {
+        "nofile": {
```

Comment:
This `nofile` section is the default. Here, I am making sure the default remains alongside the new `core` section.